appveyor.yml

```diff
@@ -22,34 +25,61 @@
   environment:
     SKIP_SASS_BINARY_DOWNLOAD_FOR_CI: true
     matrix:
+      - nodejs_version: 6
+        GYP_MSVS_VERSION: 2015
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - nodejs_version: 7
```

Comment:
Since node 6-10 use the same variables can we group them rather than duplicating them ?

Reply:
Good question, this was just pulled from the upstream. I will look at creating a variable

Reply:
Appveyor has validation and isn't liking when I add keys they don't know in the yml